pkg/neotest/chain/chain.go

```diff
@@ -171,3 +171,28 @@ func NewMultiWithCustomConfig(t *testing.T, f func(*config.ProtocolConfiguration
 	t.Cleanup(bc.Close)
 	return bc, neotest.NewMultiSigner(multiValidatorAcc...), neotest.NewMultiSigner(multiCommitteeAcc...)
 }
+
+// NewMultiWithCustomConfigNoCheck creates new blockchain instance with custom protocol
+// configuration, 4 validators and 6 committee members. Second return value is
+// for validator signer, third -- for committee. The resulting blockchain instance
+// is not running and constructor error is not checked.
+func NewMultiWithCustomConfigNoCheck(t *testing.T, st storage.Store, f func(*config.ProtocolConfiguration)) (*core.Blockchain, neotest.Signer, neotest.Signer, error) {
```

Comment:
No, because it uses non-exported neotest variables. But I've refactored this helper, so it looks a bit better now, could you take a look one more time?